src/synthesizer/emission_models/__init__.py

```diff
@@ -1,51 +1,49 @@
+from synthesizer.emission_models.agn import AGN_MODELS
```

Comment:
Sorry, new `ruff` means this all got formatted. Only changes are the renaming of `LineContinuumEmission` to `LineContributionEmission`.